src/OpenTelemetry.Exporter.Geneva/.publicApi/PublicAPI.Shipped.txt

```diff
@@ -13,6 +13,8 @@ OpenTelemetry.Exporter.Geneva.GenevaExporterHelperExtensions
 OpenTelemetry.Exporter.Geneva.GenevaExporterOptions
 OpenTelemetry.Exporter.Geneva.GenevaExporterOptions.ConnectionString.get -> string?
 OpenTelemetry.Exporter.Geneva.GenevaExporterOptions.ConnectionString.set -> void
+OpenTelemetry.Exporter.Geneva.GenevaExporterOptions.ResourceFieldNames.get -> System.Collections.Generic.IEnumerable<string!>?
+OpenTelemetry.Exporter.Geneva.GenevaExporterOptions.ResourceFieldNames.init -> void
```

Comment:
I recall, asking you to use `init`, but let’s switch to `set` instead for API consistency.

Reply:
I'll change it back